src/grpc.ts

```diff
@@ -292,7 +292,10 @@ export class GrpcClient {
   }
 
   loadProtoJSON(json: protobuf.INamespace, ignoreCache = false) {
-    const hash = objectHash(json).toString();
+    const hash = crypto
+      .createHash('md5')
```

Comment:
This code is repeated, please move it to a separate function if possible.

Reply:
use objectHash